Tweeter/Controllers/AccountController.cs

```diff
@@ -151,7 +151,7 @@ public ActionResult Register()
         {
             if (ModelState.IsValid)
             {
-                var user = new ApplicationUser { UserName = model.Email, Email = model.Email };
+                var user = new ApplicationUser { UserName = model.Username, Email = model.Username };
```

Comment:
Do you want the username saved under the email? I think you'll want to change that part back. The UserName stuff is correct though.